### PR TITLE
Individual Assignment 2b

### PR DESCRIPTION
**Pull request by AlvaroAlonso-0**

### CODE REVIEW

**Comment by ocorcho** [comment]
Well done for assignment 2.
I let @dgarijo to check the one on Python.

File: Assignment4/AlvaroAlonso-0-b190300/task07.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+# SPARQL
+for r in g.query(q1):
+  print(r.Subject)
+
+"""**TASK 7.2: List all individuals of "Person" with RDFLib and SPARQL (remember the subClasses)**"""
+q2 = prepareQuery('''
+  SELECT DISTINCT ?Subject WHERE { 
+    {?Subject rdf:type ns:Person. }
+    UNION 
+    {?s rdfs:subClassOf ns:Person.
```
Comment (dgarijo) [inline Assignment4/AlvaroAlonso-0-b190300/task07.py]: use chained properties with * to get the full taxonomy of classes.

File: Assignment4/AlvaroAlonso-0-b190300/task07.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+"""**TASK 7.3: List all individuals of "Person" and all their properties including their class with RDFLib and SPARQL**"""
+q3 = prepareQuery('''
+  SELECT DISTINCT ?Subject ?Property ?x WHERE { 
+    {?Subject rdf:type ns:Person.
+     ?Subject ?Property ?x}
+    UNION 
+    {?s rdfs:subClassOf ns:Person.
+    ?Subject rdf:type ?s.
+    ?Subject ?Property ?x}
```
Comment (dgarijo) [inline Assignment4/AlvaroAlonso-0-b190300/task07.py:78]: same as above (almost correct)

**Review by dgarijo — CHANGES_REQUESTED** [review]

File: Assignment4/AlvaroAlonso-0-b190300/task07.py (hunk 1 above)
Comment (AlvaroAlonso-0) [reply Assignment4/AlvaroAlonso-0-b190300/task07.py]: Just uploaded a new version.

**Review by dgarijo — APPROVED** [review]

**Pull request merged**